frontend/src/bundles/common/components/badge/styles.module.scss

```diff
@@ -66,3 +66,30 @@
     font-size: 10px;
     line-height: 12px;
 }
+
+.HRbadge {
```

Comment:
Since you changed the name of the prop by removing "HR", it would be better to rename the associated class name as well.

Reply:
yep, renamed